dataactvalidator/scripts/load_program_activity.py

```diff
@@ -197,3 +201,10 @@ def lowercase_or_notify(x):
 def log_blank_file():
     """ Helper function for specific reused log message """
     logger.error("File was blank! Not loaded, routine aborted.")
+
+
+def exit_if_nonlocal(exit_code):
+    if not CONFIG_BROKER['local']:
+        sys.exit(exit_code)
+    else:
+        return True
```

Comment:
Returning true would mean that none of the returns for checking `if not exit_if_nonlocal` would ever happen. Is that the intent? If not, then this isn't even needed. Just exit without checking for the else